eng/pipelines/templates/steps/analyze.yml

```diff
@@ -62,28 +70,26 @@ steps:
       node common/scripts/install-run-rush.js install
     displayName: "Install dependencies"
 
-  - template: /eng/pipelines/templates/steps/set-artifact-packages.yml
-    parameters:
-      Artifacts: ${{ parameters.Artifacts }}
-
   - script: |
-      node eng/tools/rush-runner.js build "${{parameters.ServiceDirectory}}" -packages "$(ArtifactPackageNames)" --verbose -p max
+      node eng/tools/rush-runner.js build $(ChangedServices) -packages "$(ArtifactPackageNames)" --verbose -p max
     displayName: "Build libraries"
 
   - template: /eng/pipelines/templates/steps/run-eslint.yml
     parameters:
-      ServiceDirectory: ${{ parameters.ServiceDirectory }}
+      ServiceDirectories: $(ChangedServices)
 
   - pwsh: |
-      node eng/tools/rush-runner.js check-format "${{parameters.ServiceDirectory}}" -packages "$(ArtifactPackageNames)" --verbose
+      node eng/tools/rush-runner.js check-format $(ChangedServices) -packages "$(ArtifactPackageNames)" --verbose
     displayName: "Check Format in Libraries"
 
-  - ${{ each artifact in parameters.Artifacts }}:
-      - template: /eng/common/pipelines/templates/steps/verify-changelog.yml
-        parameters:
-          PackageName: ${{artifact.name}}
-          ServiceName: ${{parameters.ServiceDirectory}}
-          ForRelease: false
+  - script: |
+      node eng/tools/rush-runner.js audit $(ChangedServices) -packages "$(ArtifactPackageNames)"
+    condition: and(succeeded(), eq(variables['RunNpmAudit'], 'true'))
+    displayName: "Audit libraries"
```

Comment:
we just removed the audit step in the latest main.

Reply:
Will do one last commit and clean it up when I touch the CChangedServices variable ref.